test/levelup-test.js

```diff
@@ -17,11 +17,11 @@ describe('levelup', function() {
 
 		zstreams
 			.fromArray(testArray)
-			.pipe(db.createWriteStream())
+			.pipe(db.createWriteStream(), { objectMode: true })
```

Comment:
The pipe() method having the ability to take these extra arguments for autoconversion MUST be very well documented.  This is the kind of thing that could easily trip someone up for hours.